Sources/MapboxMaps/Annotations/Generated/CircleAnnotationManager.swift

```diff
@@ -75,7 +82,7 @@ public class CircleAnnotationManager: AnnotationManagerInternal {
             try style.addPersistentLayer(layer, layerPosition: layerPosition)
         } catch {
             Log.error(
-                forMessage: "Failed to create source / layer in CircleAnnotationManager. Error: \(error)",
+                forMessage: "Failed to create source / layer in CircleAnnotationManager",
```

Comment:
Your changes seem to conflict with Patrick's related to ClusterOptions. Please, merge them first.